### PR TITLE
[ADD] stock_picking_transfer_lot_autoassign

### PR DESCRIPTION
**Pull request by pedrobaeza**
Auto-assignment of lots on pickings
=================================

When working with lots, it's very uncomfortable to introduce the quantity,
lot by lot, when transferring pickings from your warehouse (outgoing or
internal).

This module automatically assigns the reserved quantity as the done one, so
that you only have to change it in case of divergence, but having the
possibility of transferring directly.

Configuration
=============

* Make sure you have selected the proper removal strategy on your product
  categories.
* Configure the product on the page "Inventory", field "Tracking" with one of
  these values: "By Unique Serial Number" or "By Lots".

Usage
=====

* Create an outgoing or an internal picking.
* Include one product with lots and with enough stock.
* Click on "Mark as Todo" button, and then on "Reserve".
* Clicking on the icon with the three items bullet list on the "Operations"
  tab you will see that the quantities have been auto-assigned on the "Done"
  column.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
cc @Tecnativa

File: stock_picking_transfer_lot_autoassign/README.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================================
+Auto-assignation of lots on pickings
```
Comment (lasley) [inline stock_picking_transfer_lot_autoassign/README.rst]: Auto-assignment?

File: stock_picking_transfer_lot_autoassign/README.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+====================================
+
+When working with lots, it's very uncomfortable to introduce the quantity,
+lot by lot, when transferring pickings from your warehouse (outgoing or
+internal).
+
+This module assigns automatically the reserved quantity as the done one, so
```
Comment (lasley) [inline stock_picking_transfer_lot_autoassign/README.rst]: automatically assigns

**Review by lasley — APPROVED** [review]
Few minor comments in the readme but otherwise LGTM. I have a few customers that are going to kiss the ground you walk on for this one @pedrobaeza

**Comment by pedrobaeza** [comment]
@rafaelbn, @carlosdauden, please review

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**